Cargo.toml

```diff
@@ -15,7 +15,7 @@ edition = "2018"
 [dependencies]
 flate2 = { version = "1.0", default-features = false, optional = true }
 time = { version = "0.1", optional = true }
-podio = "0.1"
+byteorder = "1.3.4"
```

Comment:
Not: we are using “x.x” for dependencies. Any reason to use specifically version 1.3.4 instead of 1.3?